index.html

```diff
@@ -611,29 +607,14 @@ <h3>Required Context Role</h3>
 			<p class="note">An element with the appropriate <a href="#implicit_semantics">implicit WAI-ARIA semantic</a> fulfills this requirement.</p>
 		</section>
 		<section id="namecalculation">
-			<h3>Accessible Name Calculation</h3>
-			<dl class="runin">
-				<dt>Values</dt>
-				<dd>One of the following values:
-					<ol>
-						<li>author: name comes from values provided by the author in explicit markup features such as the <pref>aria-label</pref> attribute, the <pref>aria-labelledby</pref> attribute, or the host language labeling mechanism, such as the <code>alt</code> or <code>title</code> attributes in <abbr title="Hypertext Markup Language">HTML</abbr>, with HTML <code>title</code> attribute having the lowest precedence for specifying a text alternative.</li>
-						<li>contents: name comes from the text value of the <a>element</a> node. Although this may be allowed in addition to "author" in some <a>roles</a>, this is used in content only if higher priority "author" features are not provided. Priority is defined by the <a href="#mapping_additional_nd_te" class="accname">accessible name and description computation</a> algorithm [[ACCNAME-1.2]].</li>
-            <li>prohibited: the element does not support name from author. Authors MUST NOT use the <pref>aria-label</pref> or <pref>aria-labelledby</pref> attributes to name the element.</li>
-					</ol>
-				</dd>
-			</dl>
-			<section id="namecomputation">
-				<h4>Name Computation</h4>
-				<p><a href="#mapping_additional_nd_name" class="accname">Name Computation</a> is defined in the Accessible Name and Description specification.</p>
-			</section>
-			<section id="descriptioncomputation">
-				<h4>Description Computation</h4>
-				<p><a href="#mapping_additional_nd_description" class="accname">Description Computation</a> is defined in the Accessible Name and Description specification.</p>
-			</section>
-			<section id="textalternativecomputation">
```

Comment:
#textalternativecomputation, #namecomputation and #descriptioncomputation are referenced from other places (maybe outside this document too - I don't know). Where should these references be updated to?

Reply:
> Where should these references be updated to?

ouch, another oversight. They could point to accname directly, no? If they need to be reproduce, we could move this to terms.

Reply:
Fun fact: `#mapping_additional_nd_te` (which I copy&pasted from the old version) also does not seem to match an ID at this point.

Reply:
`#namecomputation` is not used in the document anymore.

Reply:
`#descriptioncomputation` is not used in the document anymore.

Reply:
`#mapping_additional_nd_description` also needs something.

Reply:
I think I fixed all relevant ones now. @jnurthen please take another look?

Reply:
@jnurthen post TPAC ping.